doc/rst/source/cookbook/gmt-latex.rst

```diff
@@ -0,0 +1,38 @@
+Using LaTeX Expressions in GMT
+==============================
+
+GMT supports the use of LaTeX equations embedded in text strings that are used
+with **-B** in titles, subtitles and axis labels, as well as for single lines
```

Comment:
Is this statement correct? For me, LaTex equations work in titles, but not subtitles.